extensions/markdown-language-features/src/tableOfContentsProvider.ts

```diff
@@ -72,7 +72,9 @@ export class TableOfContentsProvider {
 				text: TableOfContentsProvider.getHeaderText(line.text),
 				level: TableOfContentsProvider.getHeaderLevel(heading.markup),
 				line: lineNumber,
-				location: new vscode.Location(document.uri, line.range)
+				location: new vscode.Location(document.uri, new vscode.Range(
```

Comment:
If you already have all four arguments to `Range`, you can use the shorthand `Range.create(a,b,c,d)`